### ISSUE TITLE
Monorepo for all Dart packages

### ISSUE THREAD

**Issue by Vinzent03** [supabase-community/supabase-dart#96]
Let's say I have a problem with realtime there are currently 3 repositories where an issue could be tracked: `supabase-flutter`, `supabase-dart`, `realtime-dart`. This is **really** annoying. (By the way, why is `supabase-flutter` still at `supabase` and the others at `supabase-community`?). 

I think by combining them into one repo, but still separated packages, you get a better overview. You could use [Melos](https://melos.invertase.dev/) by Invertase to manage them. Versioning should be easier too.

**Comment by bdlukaa** [comment]
I once suggested this kind of approach but they said they wanted to keep the same pattern as the other client libraries for other programming languages. That means every package has an equivalent repository to postgrest, realtime, gotrue, and so on...

**Comment by Vinzent03** [comment]
I don't know what the benefit is of being like the other languages, but at least moving all issues into one repo is the next best solution.

**Comment by bdlukaa** [comment]
> I don't know what the benefit is of being like the other languages

Consistency

**Comment by dshukertjr** [comment]
>@dshukertjr and I were discussing to move all the issues to a single repo (supabase-flutter).

Right. I haven't been doing this a lot. I would have to actively start doing this.

**Comment by phamhieu** [comment]
> By the way, why is supabase-flutter still at supabase and the others at supabase-community

you are right. We should move that repo to `supabase-community` also. 

cc @kiwicopple

**Comment by Vinzent03** [comment]
I see that `supabase-flutter` is moved, but issues are still split between all repos. I would move all issues to `supabse-flutter`/ `supabase-dart` and disable issues on the rest.

**Comment by dshukertjr** [comment]
Thanks @Vinzent03 for the suggestion. 

@phamhieu I feel like client libraries of other languages might having similar problems, but have the team discussed some kind of way to centralize issues for them? Do you think we could bring up @Vinzent03 's idea of disabling issues on sub-libraries?

**Comment by bdlukaa** [comment]
I would love this too!

**Comment by Vinzent03** [comment]
Instead of disabling issues on the sub-libraries, what about adding a template which tells the user to create the issue in the `supabse-flutter` repo. When users can't directly find a way to create an issue, they might not create the issue at all or use some other ways like discord.
